crates/cast/bin/cmd/wallet/mod.rs

```diff
@@ -290,16 +294,21 @@ impl WalletSubcommands {
                     }
                 }
             }
-            Self::NewMnemonic { words, accounts, entropy } => {
+            Self::NewMnemonic { words, accounts, entropy, json } => {
+                let mut json_values = if json { Some(vec![]) } else { None };
```

Comment:
don't think we need this here, we could just have at L#341
```rust
                if json {
                    let obj = json!({
                        "mnemonic": phrase,
                        "accounts": accounts,
                    });
                    println!("{}", serde_json::to_string_pretty(&obj)?);
                }
```

Reply:
Addressed in 298346d1c9150fed581d1b73e04a658349b149e3